peer/peer.go

```diff
@@ -1348,6 +1349,13 @@ out:
 				continue
 			}
 
+			// Ignore unknown messages to ease future extensibility.
+			// Check if the error originated from wire/message.go:makeEmptyMessage().
+			if strings.Contains(err.Error(), "unhandled command") {
```

Comment:
Should turn this into a concrete error so we can use `errors.Is` here instead.

Reply:
According to:

https://github.com/btcsuite/btcd/blob/7070d53e092f1951b54cbf96f5caafadd58e6f33/wire/doc.go#L142-L146

and given that the error is returned by [`makeEmptyMessage()`](https://github.com/btcsuite/btcd/blob/7070d53e092f1951b54cbf96f5caafadd58e6f33/wire/message.go#L94) from `wire/message.go`, I guess it should be [`wire.MessageError`](https://github.com/btcsuite/btcd/blob/7070d53e092f1951b54cbf96f5caafadd58e6f33/wire/error.go#L18-L21), right? It looks like `error.Is` is not applicable to it?

Or do you think that `ErrUnhandledCommand = errors.New("unhandled command")` should be planted somewhere, e.g. in `wire/message.go` and used, probably in violation of the comment above?